tools/editor/create_dialog.cpp

```diff
@@ -107,6 +108,9 @@ void CreateDialog::add_type(const String& p_type,HashMap<String,TreeItem*>& p_ty
 
 	}
 
+        const String& description = EditorHelp::get_doc_data()->class_list[p_type].brief_description;
```

Comment:
Looks like you're using spaces for indentation while the rest of the code uses tabs, could you amend the commit to fix it?